python/ruff-ecosystem/ruff_ecosystem/defaults.py

```diff
@@ -9,6 +9,8 @@
     Repository,
 )
 
+JUPYTER_NOTEBOOK_SELECT = "A,E703,F704,B015,B018,D100"
```

Comment:
Why these?

Reply:
@dhruvmanila chose them — they're rules that have different behavior for notebooks or something? Idk if we really need to select a subset though?

Reply:
Yeah I'd probably just expand them, but nbd either way.

Reply:
Like extend select? I'll just wait for Dhruv to be available — not crtical.

Reply:
> they're rules that have different behavior for notebooks or something?

Yeah, I chose them for this exact reason but it's reasonable to just expand them and use `extend-select` if possible.